### DISCUSSION TITLE
Small contribution (FS22)

### DISCUSSION THREAD

**Discussion by AgroMeester** [Mogli12/KeyboardSteer#729] [Ideas]
Hi Stefan,

I just saw you already have a version of VCA for us to enjoy. I'm gonna try it out myself this evening.
Thanks for your work and as a small gesture a little something for you in return.

Use if you like!

Greetings,

AgroMeester

![vca-v1](https://user-images.githubusercontent.com/25507803/144103315-40d368ce-c75b-4664-8355-049dc26100a2.png)

**Comment by Mogli12** [comment]
Vielen Dank
